tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -114,4 +114,56 @@ class WskWebActionsTests
             authorizedResponse.statusCode shouldBe 200
             authorizedResponse.body().asString() shouldBe namespace
     }
+
+    it should "fail to invoke web action with an unsupported content type when x-ow-raw-http header is not supplied" in withAssetCleaner(wskprops) {
```

Comment:
I think unit tests would be sufficient here, no? No need for a full system test.